uxarray/_exodus.py

```diff
@@ -123,6 +123,7 @@ def _read_exodus(filepath, ds_var_names):
 
             # concatenate to the previous blk
             conn = np.concatenate((conn, tmp_conn))
+            conn = conn.astype(int)
```

Comment:
Instead I suggest using `np.full` instead of `np.empty` and changing dtypes

```
conn = np.full((0, max_face_nodes), 0, dtype=np.int64)
tmp_conn = np.full((value.data.shape[0], max_face_nodes), 0, dtype=conn.dtype)
```

Also why is `conn` needed It seems like `np.concatenate((conn, tmp_conn)) == tmp_conn`?

Reply:
Thanks, I've simplified things based on your input.